features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/NavBar.js

```diff
@@ -55,23 +55,23 @@ class NavBar extends Component {
                 }
                 <Divider inset={true} style={{margin: '5px'}}/>
 
-                <MenuItem style={{marginLeft: '-8px', marginTop: '10px'}} onClick={this.toggleSection}><b>THROTTLING_POLICIES</b></MenuItem>
-                {((this.state.expandedSection).includes("THROTTLING_POLICIES")) ?
+                <MenuItem style={{marginLeft: '-8px', marginTop: '10px'}} onClick={this.toggleSection}><b>THROTTLING POLICIES</b></MenuItem>
+                {((this.state.expandedSection).includes("THROTTLING POLICIES")) ?
                     (<div>
                         <MenuItem>
-                            <Link name="advanced_throttling" to="/advanced_throttling">ADVANCED THROTTLING</Link>
+                            <Link name="advanced_throttling" to="/policies/api_policies">API Rate Limits</Link>
```

Comment:
Since these are menu items, shouldn't they be in caps?

Reply:
capitalizing is a presentation option. so this is fine.

Reply:
should not we be applying the css in that case?

Reply:
Yes, by adding a `className` to the MenuItem's parent and controlling its appearance (color, size, capitalization etc.) from a related .css file